docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -0,0 +1,314 @@
+<!-- markdownlint-disable MD033 -->
+# vSphere CSI Driver - Setup Topology (Zoning)
+
+- [Set Up Zones in the vSphere CNS Environment](#set_up_zones_in_vspehre)
```

Comment:
typo in vsphere:
s/set_up_zones_in_vspehre/set_up_zones_in_vsphere

Reply:
fixed.